### DISCUSSION TITLE
How to delete a version from package manager?

### DISCUSSION THREAD

**Discussion by ramsoft-inc** [microsoft/winget-pkgs#16767] [General] [answered]
We want delete the all previous versions of packages and start with new versioning scheme.

In order to to achieve this, we tried deleting the entire folder of the previous version before I sent  merge request. But after the merge request is approved I still see the folder in repo.

Is there any thing else I should do to delete a version from repo?
Please guide me on this.

Thanks

**Comment by denelon** [comment]
@ramsoft-inc

Can you share the PR? I'll take a look.

In order to "remove" a package, each of the versions must be removed.
In order to "remove" a version of a package the manifest file(s) for that version must be removed.

After a PR has been merged, the publishing pipeline needs to complete, and update the PreIndexed package. Then that must be published to the store, and the user cache needs to be refreshed (default 5 minute interval). This process can take from half an hour to a few hours to fully propagate globally.

**Reply by ramsoft-inc** [reply]
@denelon  My PR https://github.com/microsoft/winget-pkgs/pull/15867, in which I deleted the version 2.0.0 in commit https://github.com/microsoft/winget-pkgs/pull/15867/commits/38cde7f5e2e8ae307ed63dc716de2533ef08c6a1.

**Comment by ghost** [comment, marked as answer]
I can't find any removal of packages under your username: https://github.com/microsoft/winget-pkgs/pulls?q=is%3Apr+author%3Aramsoft-inc+is%3Aclosed

**Reply by ramsoft-inc** [reply]
@ItzLevvie2 In commit https://github.com/microsoft/winget-pkgs/pull/15867/commits/38cde7f5e2e8ae307ed63dc716de2533ef08c6a1, I have deleted version 2.0.0

In my fork this folder(2.0.0) is not exist https://github.com/ramsoft-inc/winget-pkgs/tree/master/manifests/r

**Reply by ghost** [reply]
Ah I see what's happening here, there seems to be two folders of Ram**S**oft with a different capitalization of the letter `S`, so it'll show up in the repository as two folders but on GitHub Desktop as one folder.

**Reply by ramsoft-inc** [reply]
Yes, we have updated Folder name as well. Correct Folder is RamSoft (with capital S).

**Reply by ghost** [reply]
Problem should now be solved with commit https://github.com/microsoft/winget-pkgs/commit/7db458a460a546f53c39df3ed2f7a22ab52d56b7.

**Reply by ramsoft-inc** [reply]
Thank you @ItzLevvie2 for quick help.
